.storybook/assets/404.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Page not found</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+    <link rel="icon" type="image/svg+xml" href="./favicon.svg">
+    <style>
+      body {
+        --spectrum-background-layer-1-color: rgb(253, 253, 253);
+
+        margin: 0;
+        padding: 0;
+        block-size: 100vh;
+        background-color: var(--spectrum-background-layer-1-color);
+      }
+
+      main {
+        --spectrum-neutral-content-color-default: rgb(34, 34, 34);
+
+        color: var(--spectrum-neutral-content-color-default);
+        display: flex;
+        padding-inline: 20px;
+        padding-block: 4rem;
+        flex-direction: column;
+        align-items: flex-start;
+        gap: 60px;
+        max-width: 1000px;
+        margin: auto;
+      }
+
+      img {
+        max-inline-size: 100%;
+        margin-block: -20px 20px;
+      }
+
+      h1, h2, p {
+        --spectrum-sans-font-family-stack: "Nunito Sans",-apple-system,".SFNSText-Regular","San Francisco",BlinkMacSystemFont,"Segoe UI","Helvetica Neue",Helvetica,Arial,sans-serif;
+
+        -webkit-font-smoothing: antialiased;
+        font-family: var(--spectrum-sans-font-family-stack);
+      }
+
+      h1 {
+        font-weight: 700;
+        font-size: 32px;
+        margin-block: 20px 8px;
+      }
+
+      h2 {
+        font-size: 24px;
+        margin-block: 20px 8px;
+        padding-block-end: 4px;
+      }
+
+      p {
+        font-size: 14px;
+        line-height: 24px;
+        margin-block: 16px;
+      }
+
+    </style>
+  </head>
+  <body>
+    <main>
+      <h1>404</h1>
+      <section>
+        <img src="/spectrum_illustration_2x.png" alt="">
```

Comment:
Do we like this illustration (from the landing page), or instead, should we use our illustrated message component somehow?

Reply:
It looks like illustrated message needs to use an [outline illustration](https://spectrum-contributions.corp.adobe.com/page/illustrated-message-beta/#Illustration-area), so if we went that route we'd have to get a new illustration.

This one is fine! It's not thrilling or exciting, but we could potentially get some different artwork later.

Also mildly concerned that the S2 illustrated message might only support [square images](https://www.figma.com/design/eoZHKJH9a3LJkHYCGt60Vb/S2-token-specs?node-id=8096-5712&t=DSBQOW9xcsd06MQ9-4)? So it might be best not to use that at least for now until we get all of that straightened out.

Reply:
I think the illustration is okay for now, but it might be worth asking the designers if they have another image we can use that's more 404 specific and not the same one as our docs landing page.